dev/io.openliberty.org.apache.myfaces.4.0_fat/test-applications/ProgrammaticFaceletTests.war/resources/WEB-INF/beans.xml

```diff
@@ -9,8 +9,8 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
```

Comment:
Should this also be v2.0?

Reply:
Yes, fixed it under build instead of publish, along with missing server others.  They should be fixed now.